src/components/image.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Image = (props) => (
+    <div className="row">
+    {props.photos.map((image, index) => {
+      return (
+        <div key={index} className="col s4">
```

Comment:
Can be broken down into smaller components. Maybe leave out the card-content for now and create another component when you get to the story.